src/controller/connexionController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace TourCMS\OnBoarding\Controller;
+
+use Error;
+use TourCMS\OnBoarding\Model\connexion;
+
+class connexionController
+{
+    protected $connexion;
+
+    public function __construct($data = null)
+    {
+        $this->connexion = new connexion($data);
+    }
+
+    function redisDataInsertion($dataType, $data = null)
```

Comment:
Probably this function should return a true/false if the operation has happened successfully.